tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+I will come up with exercises while you review the document. 
+////
+
+//Welcome page:
+If you are completely unfamiliar with Metanorma, you’ve come to the right place. In this tutorial, you will learn how to create Metanorma documents, create outputs, and even learn the basics of troubleshooting. 
+
+Curriculum:
+// Include good-looking navigation of the headings. 
+
+== Introduction to Metanorma
+tag::[learningobjectives-1]
+
+Before we dive into using the Metanorma toolchain, let’s cover the basic ideas you need to understand to work with Metanorma. In this lesson we will cover:
+
+* What Metanorma is
+* The differences between Word and Metanorma 
+* The paradigms “What you see is what you get” vs. “What you see is what you mean”
+* What the Metanorma workflow looks like from document creation to publication
+
+end::[learningobjectives-1]
+
+=== What is Metanorma?
+include::../author/concepts/wysiwyg_vs_wysiwym.adoc[leveloffset=+2]
+include::../author/concepts/what_is_metanorma.adoc[leveloffset=+2]
+
+Let’s explore the workflow of Metanorma in the next lesson.
+>> Start next lesson //Button text
+
+=== The Metanorma Workflow
+
+include::../author/concepts/metanorma_workflow.adoc[]
+
+Let’s look at the core of each standard document: Metanorma AsciiDoc.
+>> Start next lesson //Button text
+
+
+=== Summary
+
+Let’s recap what we’ve learned so far:
+
+* Metanorma uses a what you see is what you mean (WYSIWYM) approach, meaning that the text you write is what you *want* to see.
+* You need to render documents to have a correct visible output. The appearance is controlled in a stylesheet.
+* Metanorma is an umbrella term for several things and consists of: +
+The metanorma toolchain, document models and a XML schema of these document models.
+* Metanorma documents use Metanorma AsciiDoc markup 
+* When you want to generate an output, the AsciiDoc document is converted into XML in order to be checked against the XML schema.
+* If the document is valid, Metanorma generates an HTML, PDF and Microsoft Word (`.doc`) output. 
+
+Now that we have covered the fundamentals, let’s have a look at AsciiDoc.
+>> Start next lesson
+
+== Introduction to AsciiDoc
+
+tag::[learningobjectives-2]
+
+Before we write our first Metanorma document, let us look at the foundation for all Metanorma documents: AsciiDoc. In this lesson, we will discuss what AsciiDoc is and what a typical Metanorma document looks like. You will also learn how to write AsciiDoc.
+
+end::[learningobjectives-2]
+
+Each Metanorma document is written in AsciiDoc, a markup language that annotates text with processing instructions for layout. Since AsciiDoc only consists of text, you can use any editor to create AsciiDoc files (`.adoc` files). Because Metanorma fulfills the markup requirements of SDOs, Metanorma extends plain AsciiDoc. This version of AsciiDoc is called Metanorma AsciiDoc. 
```

Comment:
```suggestion
Each Metanorma document is written in AsciiDoc, a markup language that annotates text with processing instructions for layout. Since AsciiDoc only consists of text, you can use any editor to create AsciiDoc files (`.adoc` files). AsciiDoc is very similar to Markdown or Wiki formatting but provides more precise markup for creating text documents, for example to create a bibliography section or an index.

Because Metanorma fulfills the markup requirements of SDOs, Metanorma extends plain AsciiDoc. This version of AsciiDoc is called Metanorma AsciiDoc. 
```